lib/iris/common/metadata.py

```diff
@@ -1577,46 +1456,42 @@ def values(self):
 
 
 #: Convenience collection of lenient metadata combine services.
-SERVICES_COMBINE = (
+SERVICES_COMBINE = [
     AncillaryVariableMetadata.combine,
     BaseMetadata.combine,
     CellMeasureMetadata.combine,
-    ConnectivityMetadata.combine,
     CoordMetadata.combine,
     CubeMetadata.combine,
     DimCoordMetadata.combine,
-)
+]
 
 
 #: Convenience collection of lenient metadata difference services.
-SERVICES_DIFFERENCE = (
+SERVICES_DIFFERENCE = [
     AncillaryVariableMetadata.difference,
     BaseMetadata.difference,
     CellMeasureMetadata.difference,
-    ConnectivityMetadata.difference,
     CoordMetadata.difference,
     CubeMetadata.difference,
     DimCoordMetadata.difference,
-)
+]
 
 
 #: Convenience collection of lenient metadata equality services.
-SERVICES_EQUAL = (
+SERVICES_EQUAL = [
     AncillaryVariableMetadata.__eq__,
     AncillaryVariableMetadata.equal,
     BaseMetadata.__eq__,
     BaseMetadata.equal,
     CellMeasureMetadata.__eq__,
     CellMeasureMetadata.equal,
-    ConnectivityMetadata.__eq__,
-    ConnectivityMetadata.equal,
     CoordMetadata.__eq__,
     CoordMetadata.equal,
     CubeMetadata.__eq__,
     CubeMetadata.equal,
     DimCoordMetadata.__eq__,
     DimCoordMetadata.equal,
-)
+]
```

Comment:
Ideally, we'd want to change these back to `tuple`s once `iris.experimental.ugrid` is merged into the core of `iris`.